shared/fs/popups/row-action-popup-container.js

```diff
@@ -123,11 +123,17 @@ const getRootMenuItems = (stateProps, dispatchProps) => {
       onClick: () => saveImage(path),
     })
   isMobile &&
+    pathItem.type === 'file' &&
+    // TODO: re-enable this (or refactor) when we have in-app share. This
+    // should probably be split up into separate components anyway.
```

Comment:
I thought we decided to include in-app share items in the root menu:
* Share to other app
* Share to chat conversation

Reply:
Oh; I wasn't aware of that (or I didn't remember). Will clarify in the TODO comment.